src/main/java/duke/utils/Constants.java

```diff
@@ -0,0 +1,64 @@
+package duke.utils;
+
+
+public class Constants {
+    /**
+     * Greeting for Duke.
+     */
+    public static final String GREETING =
+            "Hello/您好/こんにちは I'm Duke\n"
+            + "What can I do for you?";
+
+    /**
+     * Divider for Duke output.
+     */
+    public static final String DIVIDER = "__________________________________________________";
+
+    /**
+     * Exit pattern of the exit command for Duke.
+     */
+    public static final String EXITPATTERN = ("^(b|B)(y|Y)(e|E)$");
```

Comment:
you should use underscores to separate words! also, associated constants (final variables) should be prefixed by a common type name
eg `PATTERN_EXIT`, `PATTERN_DONE`